package.json

```diff
@@ -77,29 +77,30 @@
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/user-event": "^14.4.3",
     "@testing-library/vue": "^7.0.0",
-    "@types/lodash": "^4.14.195",
+    "@types/lodash": "^4.14.196",
     "@types/mdx": "^2.0.4",
-    "@typescript-eslint/eslint-plugin": "^5.59.11",
-    "@typescript-eslint/parser": "^5.62.0",
+    "@typescript-eslint/eslint-plugin": "^6.3.0",
+    "@typescript-eslint/parser": "^6.3.0",
     "@vitejs/plugin-vue": "^4.2.3",
-    "@vitest/ui": "^0.33.0",
-    "@vue/eslint-config-prettier": "7.1.0",
+    "@vitest/ui": "^0.34.1",
+    "@vue/eslint-config-prettier": "^7.1.0",
     "@vue/eslint-config-typescript": "^11.0.3",
     "autoprefixer": "^10.4.14",
     "chromatic": "^6.20.0",
-    "eslint": "^8.45.0",
+    "eslint": "^8.46.0",
     "eslint-plugin-prettier": "^5.0.0",
-    "eslint-plugin-vue": "^9.15.1",
+    "eslint-plugin-vue": "^9.16.1",
     "husky": "^8.0.3",
+    "jsdom": "^22.1.0",
```

Comment:
do we need `jsdom`?

Reply:
yeah, it's used by Vitest (see `vite.config.ts#test`). I think it was previously auto-installed, but after the latest update I was prompted to explicitly install it